velox/docs/functions/presto/conversion.rst

```diff
@@ -448,8 +450,22 @@ consistent with other supported cases of cast.
 
 ::
 
+  SELECT cast('infinity' as real); -- Infinity
+  SELECT cast('-infinity' as real); -- -Infinity
+  SELECT cast('inf' as real); -- Infinity
```

Comment:
presto:test> SELECT cast('inf' as real);
Query 20231120_033915_00034_a9j2g failed: Cannot cast 'inf' to REAL